tableau/1.0/modules/ROOT/pages/tableau-connector-xml-maven.adoc

```diff
@@ -0,0 +1,65 @@
+= Tableau Connector 1.0 XML and Maven Support - Mule 4
+
+Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio:
+
+* If you manually code a Mule runtime engine (Mule) app in XML, either from the Anypoint Studio XML editor or from a text editor, you can access the connector from your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
+* If you add the connector in Studio, Studio automatically populates the XML code with the connector's namespace and schema location and it also adds a dependency to the `pom.xml` file.
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/tableau
+http://www.mulesoft.org/schema/mule/tableau/current/mule-tableau.xsd
+----
+
+This example shows how the namespace statements are placed in the XML block:
+
+[source,xml,linenums]
+----
+<mule xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
+    xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:tableau="http://www.mulesoft.org/schema/mule/tableau"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core
+	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+    http://www.mulesoft.org/schema/mule/tableau
+    http://www.mulesoft.org/schema/mule/tableau/current/mule-tableau.xsd
+    http://www.mulesoft.org/schema/mule/http
+    http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+    http://www.mulesoft.org/schema/mule/ee/core
+    http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
+----
+
+== Add a POM File Dependency
+
+The Apache Maven `pom.xml` file generated by Anypoint Studio contains dependencies for a Mule app. When you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector:
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>mule4-tableau-connector</artifactId>
+  <version>x.x.x</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+Replace `x.x.x` with the version that corresponds to the connector you are using.
+
+To obtain the most up-to-date `pom.xml` file information:
+
+. Go to https://www.mulesoft.com/exchange/[Anypoint Exchange].
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for `tableau`.
```

Comment:
We can specify to search for Tableau Specialist to be safe - I don't know if the composer connectors appear in Exchange

Reply:
Yup done! It appears as Tableau Specialist in Studio so sounds good